### DISCUSSION TITLE
Update of MonarcAppFO from version 2.12.5 to version 2.12.7-p4

### DISCUSSION THREAD

**Discussion by abalis** [monarc-project/MonarcAppFO#532] [Q&A]
I have a MonarcFO installation of version 2.12.5 that I need to upgrade to version 2.12.7-p4. The installation was performed using the instructions for manual deployment on Ubuntu 22.04 (https://github.com/monarc-project/MonarcAppFO/blob/master/INSTALL/INSTALL.ubuntu2204.md). The update instructions mention that the following commands should be executed:
$ ./scripts/update-all.sh -c
$ rm -Rf ./data/cache
$ sudo systemctl restart apache2.service

But, the update-all.sh scripts requires a git project in order to perform its operations. Such git project does not exist in our installation, since the instructions of manual deployment on Ubuntu 22.04 do not include the initialization of a git project. I have tried to initialize a git project inside the installation folder in order to execute the update-all.sh script but I didn't manage to perform the upgrade. Are there any update instructions for installations created using the above instructions for Ubuntu 22.04? What are the official steps that should be followed in this case?

Thank you in advance

**Comment by ruslanbaidan** [comment]
[duplicates](https://github.com/monarc-project/MonarcAppFO/discussions/534?converting=1)
